src/DependencyInjection/SonataUserExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $loader->load('form.xml');
 
-        if (class_exists('Google\Authenticator\GoogleAuthenticator')) {
+        if (class_exists('Sonata\GoogleAuthenticator\GoogleAuthenticator')) {
```

Comment:
I think this is a BC break, maybe if you do, `if (class_exists('Google\Authenticator\GoogleAuthenticator') || class_exists('Sonata\GoogleAuthenticator\GoogleAuthenticator'))`

Reply:
or add a conflict to composer.json about the older version of library.

Reply:
not sure that adding conflict on v1 will be a good solution as it may block installation & require php7 no? think `if (class_exists('Google\Authenticator\GoogleAuthenticator') || class_exists('Sonata\GoogleAuthenticator\GoogleAuthenticator'))` is better with a message to remove the `Google` root one on next major, WDYT?

Reply:
the best way would be to add deprecations on stable and fix on master.

Reply:
done for this one @kunicmarko20 . will give a try to fix it on master